spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1466,245 +1470,216 @@ def handle_max_vel(self, request: SetVelocity.Request, response: SetVelocity.Res
             response.message = f"Error: {e}"
             return response
 
-    def _robot_command_goal_complete(self, feedback: RobotCommandFeedback) -> GoalResponse:
-        if feedback is None:
-            # NOTE: it takes an iteration for the feedback to get set.
+    def _process_full_body_command_feedback(self, feedback: FullBodyCommandFeedback) -> GoalResponse:
+        if feedback.status.value != feedback.status.STATUS_PROCESSING:
```

Comment:
This could probably use a comment - I stared at it for a bit before remembering that there's more than one status in the message...

Reply:
@abaker-bdai